pkg/cmd/pipelinerun/delete.go

```diff
@@ -70,6 +71,7 @@ or
 	f.AddFlags(c)
 	c.Flags().BoolVarP(&opts.ForceDelete, "force", "f", false, "Whether to force deletion (default: false)")
 	c.Flags().StringVarP(&opts.ParentResourceName, "pipeline", "p", "", "The name of a pipeline whose pipelineruns should be deleted (does not delete the pipeline)")
+	c.Flags().IntVarP(&opts.Keep, "keep", "", 0, "Keep N number of pipelineruns when deleting alls")
```

Comment:
```suggestion
	c.Flags().IntVarP(&opts.Keep, "keep", "", 0, "keep n number of pipelineruns when using --all flag")
```

Reply:
Would be nice in some way to indicate that ordering here is done by most recent pipelinerun.

Reply:
humm usually i would ask to my english master  👨‍🏫  (ie you 😄 )

what about ?

Keep the `last` n number of pipelineruns?

Reply:
`keep last n number of pipelineruns` sounds good to me.